src/client/resources/conversions.ts

```diff
@@ -55,12 +59,16 @@ export function createMigCluster(
   };
 }
 
-export function updateMigClusterUrl(
+export function updateMigCluster(
   clusterUrl: string,
+  insecure: boolean,
+  caBundle: string,
 ) {
   return {
     spec: {
       url: clusterUrl,
+      insecure,
+      caBundle: caBundle || null,
```

Comment:
The backend returns a 422 if this value is the empty string, since it's apparently expecting either null or a base64 encoded string that it can fit into a `[]byte`. Fixing it this way seems sort of ugly, not sure if there's a better approach.